### ISSUE TITLE
Use options with pretty-simple as default ghci printer

### ISSUE THREAD

**Issue by nasosev** [cdepillabout/pretty-simple#108]
I have pretty-simple set as the default printer for my project by putting the following in my .ghci file at the root of the project:

```
:set -ignore-package pretty-simple -package pretty-simple
:def! pretty \ _ -> pure ":set -interactive-print Text.Pretty.Simple.pPrint"
:pretty
```

How can I also enable the compact output option for the default printer?

Thanks a lot.

**Comment by georgefst** [comment]
Yeah, I'd like to flesh out that section of documentation a little. I started looking in to it properly once, but the whole process of writing .ghci files is fraught with awkward edge cases. Especially when needing to import packages. For one thing, I've never been clear about the rules around qualifying imports. I eventually got hung up on https://github.com/haskell/cabal/issues/7789. There are definitely upstream improvements to be made, in GHC as well as Cabal.

Anyway, this works for me on GHC 9.2:
```
import Text.Pretty.Simple
ghciInteractivePrint = pPrintOpt CheckColorTty defaultOutputOptionsDarkBg{outputOptionsCompact = True}
:set -interactive-print ghciInteractivePrint
```

**Comment by nasosev** [comment]
Thank you. That works for me if I input it at the terminal, but it does not work automatically when I put it in my `.ghci`.

**Comment by scarf005** [comment]
I tried the [above comment's solution](https://github.com/cdepillabout/pretty-simple/issues/108#issuecomment-1092608377) and set up `~/.ghci` to this:

```hs
import Text.Pretty.Simple

ghciInteractivePrint = pPrintOpt CheckColorTty defaultOutputOptionsDarkBg{outputOptionsCompact = True}
:set -interactive-print ghciInteractivePrint
```

![image](https://user-images.githubusercontent.com/54838975/232259869-2544e665-414b-4c70-8996-4fd95b9441ee.png)

however it doesn't seem to work and I have to manually type them to work. could anyone help me figure out what's the problem?

**Comment by georgefst** [comment]
Yeah, I've ended up often using `cabal repl -b pretty-simple --repl-options="-interactive-print Text.Pretty.Simple.pPrintForceColor"` because the `.ghci` solution doesn't always work (see https://github.com/haskell/cabal/issues/7789).
